

class ReprocessingError(Exception):
    pass


class DataError(ReprocessingError):
    pass